src/backend/middleware/index.ts

```diff
@@ -41,7 +41,7 @@ export function middleware(kojiConfig: KojiConfig = {}) {
     // Use the remixData from the json configuration as the base
     const { remixData = {} } = kojiConfig;
 
-    // Apply remix-specific overrides (equivalent of window.KOJI_OVERRIDES.overrides on the client)
+    // Apply customization-specific overrides (equivalent of window.KOJI_OVERRIDES.overrides on the client)
```

Comment:
```suggestion
    // Apply version-specific overrides (equivalent of window.KOJI_OVERRIDES.overrides on the client)
```

Again, "customization" isn't quite right, but not sure my fix is either.

Reply:
Went with "version-specific". It's an internal comment anyway, so not high priority.